### PR TITLE
Refactored PowerNet

### PR DESCRIPTION
**Pull request by 778b**
## About the PR 
Just some refactoring thing, no big code changes, no screenshots.

**Changelog**

Closes https://github.com/space-wizards/space-station-14/issues/12353

- Removed all local call IoCManager.Resolver in PowerNet.cs
- Added a default private field with IoCManager.Resolver in PowerNet.cs

### CODE REVIEW

**Comment by Just-a-Unity-Dev** [comment]
CL should only show playerfacing changes

**Comment by 778b** [comment]
> CL should only show playerfacing changes

Now its ok?

**Comment by Just-a-Unity-Dev** [comment]
yep

File: Content.Server/Power/NodeGroups/PowerNet.cs [hunk 1]
```diff
@@ -29,6 +29,7 @@ public interface IPowerNet : IBasePowerNet
     public sealed class PowerNet : BaseNetConnectorNodeGroup<IPowerNet>, IPowerNet
     {
         private PowerNetSystem? _powerNetSystem;
+        private readonly IEntityManager _entMan = IoCManager.Resolve<IEntityManager>();
```
Comment (metalgearsloth) [inline Content.Server/Power/NodeGroups/PowerNet.cs:32]: You shouldn't do IoC resolves in private fields because it causes threading issues.

**Review by metalgearsloth — DISMISSED** [review]

**Pull request merged**